src/Compilers/Core/Portable/Symbols/TypeCompareKind.cs

```diff
@@ -25,6 +25,7 @@ internal enum TypeCompareKind
         IgnoreNullableModifiersForReferenceTypes = 8,
         ObliviousNullableModifierMatchesAny = 16,
         IgnoreNativeIntegers = 32,
+        FunctionPointerRefMatchesOutInRefReadonly = 64,
```

Comment:
>FunctionPointerRefMatchesOutInRefReadonly [](start = 8, length = 41)

Consider adding a doc comment, including a brief explanation when is this needed.

Reply:
Also, it feels strange to have an "ignore" option that is not part of ```AllIgnoreOptions```. Might want to explain this as well.

---
In reply to: [426085624](https://github.com/dotnet/roslyn/pull/44257#discussion_r426085624) [](ancestors = 426085624)